setup.py

```diff
@@ -24,7 +27,6 @@
     tests_require=["nose"],
     install_requires=[
         "future==0.18.2",
-        "pycryptodome==3.10.1",
-        "urllib3[secure]==1.26.4",
+        "urllib3[secure]==1.26.6",
```

Comment:
```
    install_requires=[
        "future==0.18.2",
        "urllib3==1.26.6",
        "certifi==2021.5.30",
    ],
```
removing the [secure] and adding a requirement of certifi makes this actually installable.

We can go back to urllib3[secure] after they get their install act together (or perhaps in April 2023 when Ubuntu 18 LTS ends Standard Support)